cloudbuild.yaml

```diff
@@ -4,6 +4,7 @@ options:
   machineType: 'E2_HIGHCPU_32'
   env:
     - DOCKER_CLI_EXPERIMENTAL=enabled
+    - BUILDX_NO_DEFAULT_ATTESTATIONS=1 
```

Comment:
What is this?

Reply:
Without this multi-arch images creates two additional image with default attestation. 
https://screenshot.googleplex.com/8aaumgjd3o9h2hL

Reply:
What are the default attestations? Do we not want them and will removing them cause any issues?